test/inc/TestDef.h

```diff
@@ -14,3 +15,34 @@ static const std::wstring c_testFileExtension_Packaged = L".reuniontestfile-pack
 static const std::wstring c_testFilePhaseEventName = L"ReunionTestFilePhaseEventName";
 
 static const std::wstring c_testStartupPhaseEventName = L"ReunionTestStartupPhaseEventName";
+
+static const std::wstring c_testInstanceRedirectedPhaseEventName = L"ReunionTestInstanceRedirectedPhaseEventName";
+
+#ifndef WIDEN2
+#define WIDEN2(x) L ## x
+#endif
+
+#ifndef WIDEN
+#define WIDEN(x) WIDEN2(x)
+#endif
```

Comment:
Does STL have something for this?

#x is accomplished via
```
// Rely on _STRINGIZE(x) in yvals_core.h
#ifndef _STRINGIZE
#define _STRINGIZEX(x)  #x
#define _STRINGIZE(x)   _STRINGIZEX(x)
#endif
```

Is there something equivalent already present in std headers to do this?